src/plugins/query_enhancements/public/query_assist/utils/create_extension.tsx

```diff
@@ -147,5 +214,19 @@ const QueryAssistWrapper: React.FC<QueryAssistWrapperProps> = (props) => {
   }, [props]);
 
   if (!visible) return null;
-  return <>{props.children}</>;
+  return (
+    <>
+      <QueryAssistContext.Provider
+        value={{
+          question,
+          question$,
```

Comment:
Actually, `question$` is always created, it could be undefined because for `QueryAssistBanner`, it doesn't use this variable, so I think we don't need to pass `question$` to it thus it's undefined